app/views/doctors/transcripts/index.haml

```diff
@@ -1,5 +1,4 @@
 - if @transcripts.present?
```

Comment:
You should add a title to the page and provide a message if the doctor doesn't have any transcripts